powersimdata/data_access/profile_helper.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+import requests
+from tqdm.auto import tqdm
+
+from powersimdata.utility import server_setup
+
+
+class ProfileHelper:
+    BASE_URL = "https://bescienceswebsite.blob.core.windows.net/profiles"
+
+    @staticmethod
+    def get_file_components(scenario_info, field_name):
+        version = scenario_info["base_" + field_name]
+        file_name = field_name + "_" + version + ".csv"
+        grid_model = scenario_info["grid_model"]
+        from_dir = f"raw/{grid_model}"
+        return file_name, from_dir
+
+    @staticmethod
+    def download_file(file_name, from_dir):
+        print(f"--> Downloading {file_name} from blob storage.")
+        url = f"{ProfileHelper.BASE_URL}/{from_dir}/{file_name}"
+        dest = os.path.join(server_setup.LOCAL_DIR, from_dir, file_name)
+        os.makedirs(os.path.dirname(dest), exist_ok=True)
+        resp = requests.get(url, stream=True)
+        content_length = int(resp.headers.get("content-length", 0))
+        with open(dest, "wb") as f:
+            with tqdm(
+                unit="B",
+                unit_scale=True,
+                unit_divisor=1024,
+                miniters=1,
+                total=content_length,
+            ) as pbar:
+                for chunk in resp.iter_content(chunk_size=4096):
+                    f.write(chunk)
+                    pbar.update(len(chunk))
+
+        print("--> Done!")
+        return dest
+
+    @staticmethod
+    def parse_version(grid_model, kind, version):
+        """Parse available versions from the given spec
+
+        :param str grid_model: grid model.
+        :param str kind: *'demand'*, *'hydro'*, *'solar'* or *'wind'*.
+        :param dict version: json response
+        :return: (*list*) -- available profile version.
+        :raises ValueError: if kind not one of *'demand'*, *'hydro'*, *'solar'* or
+            *'wind'*.
+        """
+        profile_kind = {"demand", "hydro", "solar", "wind"}
+        if kind not in profile_kind:
+            raise ValueError("kind must be one of %s" % " | ".join(profile_kind))
```

Comment:
Not sure we need this check since it is already done in the `get_data` method of the `InputData` class:
```
if field_name in profile_kind:
    helper = ProfileHelper
```